packages/components/components/elvis-datepicker/src/react/elvia-datepicker.tsx

```diff
@@ -54,6 +55,23 @@ export const Datepicker: React.FC<DatepickerProps> = ({
   inlineStyle,
   webcomponent,
 }) => {
+  const lang = useLanguage();
+
+  const labels =
+    lang === 'no'
+      ? {
+          clearButtonText: clearButtonText ?? 'Nullstill',
+          label: label ?? 'Velg dato',
+          placeholder: placeholder ?? 'dd.mm.åååå',
+          openDatepicker: 'Åpne datovelger',
+        }
+      : {
+          clearButtonText: clearButtonText ?? 'Clear',
```

Comment:
Note that we've translated 'nullstill' as 'clear' here, but in `elvis-datepicker/src/react/calendar/calendar.tsx`, we translated it as 'reset.'

I recommend choosing one term and using it consistently in both places. Either 'clear' or 'reset' works well. Just ensure that we're consistent 🙂.